src/Core/Configurations/ConfigFileWatcher.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
why is this class being shown as newly added, this was already a part of your last PR that got merged.